app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) < 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format. Use"
+                         " 'mv source_path destination_path'.")
+
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if not os.path.isfile(source_path):
+        raise FileNotFoundError(f"Source file '{source_path}' "
+                                f"does not exist.")
+
+    directory_path = os.path.dirname(destination_path)
```

Comment:
Use `os.path.join` to concatenate parts of the path. This will make your app cross-platform and handle path separators correctly, as suggested in the checklist.